GermanFilter/sections/specific.txt

```diff
@@ -67,6 +67,8 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
 !
 radiosaw.de###oms_gpt_rectanglesidebar
 fitbook.de##a[href^="https://secure.adnxs.com/"]
+||hasepost.de/wp-content/banners/
+||hasepost.de/fpassage/aktuell.gif
```

Comment:
Not shown.

Reply:
for example <a href="https://www.hasepost.de/zahl-der-aktuell-corona-infizierten-region-osnabrueck-ruecklaeufig-181882/">here</a>
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/77898498-eb2b7f00-7283-11ea-91a9-bcb87999eaec.png">
</details>